test/simulator/timestamp/call_log_manager/call_log_manager.cpp

```diff
@@ -88,13 +88,13 @@ call_log_manager::process_call_log_entry(json call_log_entry)
 
         switch (_api_map.at(method_name)) {
         case api_method::begin_transaction: {
-            session_simulator *session = get_session(call_log_entry);
+            session_simulator *session = get_session(call_log_entry["session_id"].get<std::string>());
```

Comment:
instead of directly passing it in, we should do what you did [here](https://github.com/wiredtiger/wiredtiger/pull/8161/commits/610696bf91af2caa6264d90c43062863b02ea3b6#diff-7ab6317a6b539815357734f932d3bd74e6d86837e1f4288774b0153eb879f98aR96)